app/build.gradle.kts

```diff
@@ -0,0 +1,173 @@
+import com.android.build.gradle.api.ApplicationVariant
+import com.android.build.gradle.api.BaseVariantOutput
+
+plugins {
+    id(Config.ApplyPlugins.ANDROID_APPLICATION)
+    kotlin(Config.ApplyPlugins.Kotlin.ANDROID)
+    id(Config.ApplyPlugins.KSP)
+    id(Config.ApplyPlugins.PARCELIZE)
+}
+
+extra.set("jacocoCoverageThreshold", 0.40.toBigDecimal()) // module specific code coverage verification threshold
+
+apply(from = "../renameAppBundle.gradle.kts") // configures additional gradle tasks to rename app bundles (when needed)
+
+// Prep BuildInfoManager to use its functions/properties later throughout this build script
+BuildInfoManager.initialize(
+    BuildInfoInput(
+        appVersion = AppVersion(major = 1, minor = 0, patch = 0, hotfix = 0, showEmptyPatchNumberInVersionName = true), // TODO: TEMPLATE - Replace with appropriate app version
```

Comment:
@Josh-Gebbeken nit: As this is just a demo for maps, you should be able to remove all of the `// TODO: TEMPLATE` comments from the project